app/templates/client/ng1/dev/app.config.js

```diff
@@ -5,11 +5,9 @@
     .config([
       "$locationProvider",
       function($locationProvider) {
-        <% if (!cordova) { %>
-        $locationProvider.html5Mode(true);
-        <% } else {%>
-        $locationProvider.html5Mode(true);
```

Comment:
The second, because the first one would have a space between `$locationProvider` and its methods.